config/locales/en.yml

```diff
@@ -211,7 +211,6 @@ en:
       organisation_settings: Organisation settings
       organisation_permissions: Organisation permissions
       change_organisation_permissions: Change permissions
-      org_permissions: Organisational permissions
```

Comment:
The word "organisational" still appears in the project, although that looks to be only in relation to user level permissions (in manage and support). I think that can be addressed in the next chunk of work

Reply:
yea, we should probably add tickets for the support ones just incase

Reply:
https://trello.com/c/pQDycRli/1813-remove-mentions-of-organisational-permissions-from-support Support card here